pkg/vadmin/opts/addnode/opts.go

```diff
@@ -25,6 +25,8 @@ type Parms struct {
 	InitiatorIP   string
 	Hosts         []string
 	Subcluster    string
+	// will be removed after VER-88096
+	Nodes map[string]string
```

Comment:
It may be temporary, but it would be good to include a more descriptive name. How about renaming it to something like this?
```suggestion
	VNodesToHostMap map[string]string
```